web-frontend/src/mirage/models/index.ts

```diff
@@ -6,8 +6,10 @@ import { TransferModel } from '../../models/TransferModel';
 
 const accountModel: ModelDefinition<AccountObject> = Model.extend({});
 const transferModel: ModelDefinition<TransferModel> = Model.extend({});
+const assetModel: ModelDefinition<TransferModel> = Model.extend({});
```

Comment:
Model for asset is the same as transfer? 🤔

Reply:
noooo oopps